### ISSUE TITLE
Error on a11y:contactEmail

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/epubcheck#1669]
Hi,
EPUBCheck returns an error when you enter the metadata:

```xml
<meta property="a11y:contactEmail">
accessibility@example.com
</meta>
```

Reference: https://www.w3.org/TR/epub-a11y-12/#contactEmail

Gregorio

**Comment by rdeltour** [comment]
Thanks for the report Gregorio!
The next release will allow the property. Note however that its value will not be checked as a valid email address (I'm waiting on further refactoring of the property processing code); but at least using the property will no longer cause a false-positive.
